### PR TITLE
Remove translations not used by Peatio (closes #714)

### LINKED ISSUE #714: Remove translations not used by Peatio

### PR DESCRIPTION
**Pull request by ysv**

### CODE REVIEW

**Comment by yivo** [comment]
@yssavchuk Tests are failing :(

**Comment by yivo** [comment]
@yssavchuk please 1) rebase; 2) update russian lang locales

**Comment by ysv** [comment]
@yivor rebased russian locales are up to date

File: config/locales/server.en.yml [hunk 1]
```diff
@@ -473,120 +473,23 @@ en:
...
         full_ask: EMPTY
       order_book:
-        title: Order Book
-        volume: Volume
-        price: Price
-        amount: Amount
-        ask: ASK
-        bid: BID
-        fade_toggle_depth: Market Depth
+        latest_trade: Latest Trade
```
Comment (yivo) [inline config/locales/server.en.yml:486]: @yssavchuk Can you explain what is this?

File: config/locales/server.en.yml [hunk 2]
```diff
@@ -473,120 +473,23 @@ en:
...
-        time: Time
-        price: Price
-        volume: Volume
-      header:
-        network: Connection State
-        vol: 24h Volume
-        high: High
-        low: Low
-        change: Change
       settings:
```
Comment (yivo) [inline config/locales/server.en.yml:490]: @yssavchuk I don't think these vars are used on peatio...

**Review by yivo — CHANGES_REQUESTED** [review]

File: config/locales/server.en.yml (hunk 2 above)
Comment (yivo) [reply config/locales/server.en.yml:490]: @yssavchuk I reviewed this code. Seems like stupid chine coders put some transactions in JS file and some in gon vars...

**Review by yivo — CHANGES_REQUESTED** [review]

**Review by yivo — APPROVED** [review]

**Pull request merged**